src/importexport/guitarpro/internal/gtp/gpconverter.cpp

```diff
@@ -263,7 +263,7 @@ void GPConverter::addBarline(const GPMasterBar* mB, Measure* measure)
     }
 
     GPMasterBar::TimeSig sig = mB->timeSig();
-    auto scoreTimeSig = Fraction(sig.enumerator, sig.denumerator);
+    auto scoreTimeSig = Fraction(sig.enumerator, sig.denomerator);
```

Comment:
Sigh, this isn't a proper correction. It's a hazard of substrings and sorting, I need to change my tooling to sort longer matches before shorter matches -- but right now I use Google Sheets and just let it do sorting, but it clearly favors `denom` over `denominator`.